xbmc/platform/android/activity/XBMCApp.cpp

```diff
@@ -251,8 +251,11 @@ void CXBMCApp::onResume()
     RequestVisibleBehind(true);
 
   if (g_application.IsInitialized())
-    dynamic_cast<CAndroidPowerSyscall*>(CServiceBroker::GetPowerManager().GetPowerSyscall())
-        ->SetOnResume();
+  {
+    IPowerSyscall* syscall = CServiceBroker::GetPowerManager().GetPowerSyscall();
+    if (syscall)
+      dynamic_cast<CAndroidPowerSyscall*>(syscall)->SetOnResume();
```

Comment:
should be able to use static_cast here and below?

Reply:
seems reasonable